admin/model_data_lake_store_settings.go

```diff
@@ -23,10 +23,8 @@ type DataLakeStoreSettings struct {
 	Prefix *string `json:"prefix,omitempty"`
 	// Flag that indicates whether the bucket is public. If set to `true`, MongoDB Cloud doesn't use the configured AWS Identity and Access Management (IAM) role to access the S3 bucket. If set to `false`, the configured AWS IAM role must include permissions to access the S3 bucket.
 	Public *bool `json:"public,omitempty"`
-	// Physical location where MongoDB Cloud deploys your AWS-hosted MongoDB cluster nodes. The region you choose can affect network latency for clients accessing your databases. When MongoDB Cloud deploys a dedicated cluster, it checks if a VPC or VPC connection exists for that provider and region. If not, MongoDB Cloud creates them as part of the deployment. MongoDB Cloud assigns the VPC a CIDR block. To limit a new VPC peering connection to one CIDR block and region, create the connection first. Deploy the cluster after the connection starts.
+	// Microsoft Azure Regions.
```

Comment:
@wtrocki (for when you return)

Maybe this decision has been captured elsewhere but if the property is shared between several models (OneOf), do we have a means by which the description is selected?

Having a default for this scenario would be great for documentation purposes (maybe worth considering for a proposal in the [openapi contract enhancements epic](https://jira.mongodb.org/browse/CLOUDP-192035))

[for reference](https://github.com/mongodb/atlas-sdk-go/pull/159/files#r1294863219)

Reply:
> do we have a means by which the description is selected?

Selection of description makes no sense. Currently we default to the last description which seems wrong.
IMHO we should:

1) Merge all descriptions into one. 
2) Remove descriptions

My preference is the 1 so I will make PR that will introduce that change.